Linux compilation issue with GCC 9.2.1

I get these errors when trying to compile PCS2. GCC 9.2.1
```
/home/tomi/PCS2/src/geo_sphere.cpp: In member function ‘ogl_vertex_buffer geo_sphere::make_vertex_buffer(int)’:
/home/tomi/PCS2/src/geo_sphere.cpp:189:2: error: ‘glGenBuffersARB’ was not declared in this scope
  189 |  glGenBuffersARB(1, &ret.buffer);
      |  ^~~~~~~~~~~~~~~
/home/tomi/PCS2/src/geo_sphere.cpp:191:2: error: ‘glBindBufferARB’ was not declared in this scope
  191 |  glBindBufferARB(GL_ARRAY_BUFFER_ARB, ret.buffer);
      |  ^~~~~~~~~~~~~~~
/home/tomi/PCS2/src/geo_sphere.cpp:192:2: error: ‘glBufferDataARB’ was not declared in this scope
  192 |  glBufferDataARB(GL_ARRAY_BUFFER_ARB, tri.size()*sizeof(vector3d),NULL, GL_STATIC_DRAW_ARB);
      |  ^~~~~~~~~~~~~~~
/home/tomi/PCS2/src/geo_sphere.cpp:193:23: error: ‘glMapBufferARB’ was not declared in this scope
  193 |  float*map = (float *)glMapBufferARB(GL_ARRAY_BUFFER_ARB, GL_WRITE_ONLY_ARB);
      |                       ^~~~~~~~~~~~~~
/home/tomi/PCS2/src/geo_sphere.cpp:196:3: error: ‘glUnmapBufferARB’ was not declared in this scope; did you mean ‘glXSwapBuffers’?
  196 |   glUnmapBufferARB(GL_ARRAY_BUFFER_ARB);
      |   ^~~~~~~~~~~~~~~~
      |   glXSwapBuffers
make[2]: *** [src/CMakeFiles/pcs2.dir/build.make:180: src/CMakeFiles/pcs2.dir/geo_sphere.cpp.o] Błąd 1
make[1]: *** [CMakeFiles/Makefile2:132: src/CMakeFiles/pcs2.dir/all] Błąd 2
make: *** [Makefile:84: all] Błąd 2

```

--- a/src/GLee.h
+++ b/src/GLee.h
@@ -62,6 +62,7 @@
 #else // GLX
 	#define __glext_h_  /* prevent glext.h from being included  */
 	#define __glxext_h_ /* prevent glxext.h from being included */
+	#define GL_GLEXT_PROTOTYPES
 	#define GLX_GLXEXT_PROTOTYPES
 	#include <GL/gl.h>
 	#include <GL/glx.h>

